src/lib/services/map/providers/google/provider.ts

```
@@ -0,0 +1,178 @@
+import type {
```

ESLint

[error] Insert `import·config·from·'$lib/config';⏎import·type·{Location}·from·'$lib/interfaces/location';⏎`

(prettier/prettier)